assets/src/styles/components/_water-level-table.scss

```diff
@@ -11,7 +11,7 @@
     }
   }
   .active {
-    background-color: $color-gray-lighter;
+    background-color: color('gray-20');
```

Comment:
So we should not be using color variables or screen size variables? Should we be using any USWDS variables?

Reply:
It seems variables would primarily be used for custom theming. I'm not sure if we'd be reading them much, or just overriding defaults.